client/extensions/woocommerce/state/action-types.js

```diff
@@ -13,6 +13,8 @@ export const WOOCOMMERCE_PRODUCT_CREATE = 'WOOCOMMERCE_PRODUCT_CREATE';
 export const WOOCOMMERCE_PRODUCT_CREATE_SUCCESS = 'WOOCOMMERCE_PRODUCT_CREATE_SUCCESS';
 export const WOOCOMMERCE_PRODUCT_EDIT = 'WOOCOMMERCE_PRODUCT_EDIT';
 export const WOOCOMMERCE_PRODUCT_VARIATION_EDIT = 'WOOCOMMERCE_PRODUCT_VARIATION_EDIT';
+export const WOOCOMMERCE_SETUP_CHOICES_REQUEST = 'WOOCOMMERCE_SETUP_CHOICES_REQUEST';
```

Comment:
LOL. I tried. English is hard. Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/5512a8530a3394de9a0cedd0beb566762edddebc